### PR TITLE
Migrate webapi and console projects 2/2

### PR DESCRIPTION
**Pull request by peter-csala**
# Description
## `PollyDemos` related changes
- It turned out the `AddSummary` methods are not needed so, I've deleted them

## `PollyTestClientDemo` related changes
- Updated the bulkhead demo calls to concurrency limiter demo calls 

## `PollyTestWebApi` related changes
- Deleted every controllers except the `ValueController`
  - `NonThrottledFaulting` and `NonThrottledGood` are replaced with minimal API
  - `DocumentationController` is pointless IMHO

### CODE REVIEW

File: PollyTestClientConsole/Program.cs
[the remark this answers is not in the document]
Comment (peter-csala) [reply PollyTestClientConsole/Program.cs:1]: You are right, let me correct it quickly.

[the remark this answers is not in the document]
Comment (peter-csala) [reply PollyTestClientConsole/Program.cs:1]: Done :)

**Comment by peter-csala** [comment]
@martincostello Is there any other thing that needs to be done prior merging it?

**Comment by martincostello** [comment]
I don't quite follow why the async demo is still changed in the diff, it should be back as it was to 00 too, no?

**Comment by peter-csala** [comment]
> I don't quite follow why the async demo is still changed in the diff, it should be back as it was to 00 too, no?

Only a single demo should be uncommented. In the part 1 I've (accidentally) committed a change where I used AsyncDemo. That's why github shows it as a change.

**Comment by martincostello** [comment]
Rather than have it uncommented which just looks messy, can you add a boolean people can change which is like `bool useAsyncDemos = true|false;` and then add an `if-else` block around the two sets?

**Comment by peter-csala** [comment]
> Can we at least do a basic tidy up on it before going further on whatever fancy libraries we could use instead?

I hope [this is](https://github.com/App-vNext/Polly-Samples/blob/1da56ac61567b59e73f7c79f222069a37f15091b/PollyTestClientConsole/Program.cs#L35) what you meant.

**Review by martincostello — APPROVED** [review]

**Pull request merged**